cypress/support/commands.js

```diff
@@ -36,4 +36,45 @@
 //     .then(function(){
 //       log.snapshot().end()
 //     })
-// })
\ No newline at end of file
+// })
+Cypress.addParentCommand("signUp", function(){
+
+  var USERNAME   = "NikolaCar"
+  var EMAIL   = "nikola@car.com"
+  var PASSWORD = "nikolacar"
+  var PASSWORD_CONFIRMATION = "nikolacar"
+  var cmd = Cypress.Log.command({
+    name: "fill the form",
+    message: [],
+    onConsole: function(){
+      return {
+        "Inserted values": [USERNAME ,EMAIL, PASSWORD, PASSWORD_CONFIRMATION]
+      }
+    }
+  })
+
+  cy
+    .chain()
+
+    .get('input[type="text"]:first', {log: false})
+      .type(USERNAME , {log: false})
+    .get('input[type="text"]:last', {log: false})
+      .type(EMAIL , {log: false})
+    .get('input[type="password"]:first', {log: false})
+      .type(PASSWORD , {log: false})
+    .get('input[type="password"]:last', {log: false})
+      .type(PASSWORD_CONFIRMATION , {log: false})
+    .get('select').select('(GMT-11:00) Pago Pago')
+    .end()
```

Comment:
To kontam nego sam namerno ostavio tako da zakomplikujem da bi teze nasao input :)